{{cookiecutter.repo_name}}/setup.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# pylint: disable=C0111,W6005,W6100
+# pylint: disable=C0111
```

Comment:
Huh, do we really not need the W6100 suppression anymore?  That's the one from `caniusepython3` that complains about string literals without `b` or `u` prefixes when not using `from __future__ import unicode_literals`.  And `setup.py` doesn't work well in Python 2 with unicode literals or Python 3 with byte literals.